src/libraries/System.Text.Json/tests/TrimmingTests/System.Text.Json.TrimmingTests.proj

```diff
@@ -1,5 +1,64 @@
 <Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
 
+  <ItemGroup>
```

Comment:
I think this drops the `StackOrQueueNotRootedTests.cs`. You either need to have all files explicitly included, or none.

@joperezr - maybe we should re-think how we glob the test files so this kind of mistake doesn't happen. What if we used a convention that each test case ended with `Test.cs` (or a exclude pattern like `.NotTest.cs`), and we only included test files by default? Then when a test wanted to include an extra file, it could use the MSBuild `Update` syntax:

```xml
    <TestConsoleAppSourceFiles Update="SerializerEntryPoint\Deserialize.FromReader.BoxedObject.cs">
      <AdditionalSourceFiles>Helper.cs</AdditionalSourceFiles>
    </TestConsoleAppSourceFiles>
```

Thoughts?

Reply:
That sounds good to me. Honestly the original thought was to move to explicitly adding them to the projects and stop doing globbing in order to allow customization and also to mimic what we do in the rest of the repo where we by default always disable glob adding of source files, but I decided to support both modes in order to not conflict my PR with other PRs from you and @layomia which were open at the same time. All that said, if we want to continue the support for glob adding tests, your suggestion sounds great to me, or if we instead want to mimic the rest of the repo and do explicit includes only, I'm fine with that too, both are very trivial changes.

Reply:
I'm in favor of requiring that the files be explicitly added, so that we are more consistent with the default behavior in the rest of the repo. I'll make a PR for this if there's no opposition to this.

Reply:
sounds good to me